Source/Details/ASRangeController.mm

```diff
@@ -399,7 +400,7 @@ - (void)_updateVisibleNodeIndexPaths
   if (ASDisplayNode.shouldShowRangeDebugOverlay) {
     ASScrollDirection scrollableDirections = ASScrollDirectionUp | ASScrollDirectionDown;
     if ([_dataSource isKindOfClass:NSClassFromString(@"ASCollectionView")]) {
-        scrollableDirections = ((ASCollectionView *)_dataSource).scrollableDirections;
+        scrollableDirections = ((ASCollectionView *)_dataSource).collectionNode.scrollableDirections;
```

Comment:
I agree that we should fix this warning. However, given that this is a legitimate usage of the collection view's property, I don't think this is the best fix. Another way to fix it is simply importing `ASCollectionView+Undeprecated.h`. We have done this elsewhere in the framework.